components/create_comment/create_comment.jsx

```diff
@@ -356,7 +360,7 @@ class CreateComment extends React.PureComponent {
     }
 
     setCaretPosition = (newCaretPosition) => {
-        const textbox = this.refs.textbox.getInputBox();
+        const textbox = this.textboxRef.current.getInputBox();
```

Comment:
given that throughout the file it is always checking if the reference is set, I'd be cautious here and do the same:
```suggestion
        const textbox = this.textboxRef.current && this.textboxRef.current.getInputBox();
```